prometheus-to-sd/main.go

```diff
@@ -259,3 +269,32 @@ func parseMonitoredResourceLabels(monitoredResourceLabelsStr string) map[string]
 	}
 	return labels
 }
+
+func getEndpoint(input string) (string, error) {
```

Comment:
add unit test

Reply:
where do you suggest we put these functions to? we can't have `main_test.go`, can we?

Reply:
I think it's ok to not have unit test for this method because:
1. we've manually verified it with https://go.dev/play/p/pMP9_m2ZWXK
2. this is during startup so future issues in URL parsing will be easily catched by e2e test

Reply:
Ok.